apps/connections/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "ipfs-webui-connections",
+  "version": "0.3.0",
+  "description": "A frontend for IPFS",
+  "main": "index.jsx",
+  "dependencies": {
+    "bootstrap": "^3.3.4",
+    "font-awesome": "^4.3.0",
+    "ipfs-web-app": "^0.1.0",
+    "react": "0.13.1",
+    "three": "^0.71.0"
+  },
+  "devDependencies": {
+    "browserify": "10.2.3",
+    "less": "2.5.0",
+    "pre-commit": "1.0.6",
+    "reactify": "1.1.1",
+    "standard": "3.3.2"
+  },
+  "ipfs-web-app": {
+    "displayname": "Connections",
+    "sort": 1,
+    "icon": "fa-globe",
+    "build": {
+      "dist/script.js": "browserify -t reactify index.jsx",
+      "dist/style.css": "lessc style/style.less"
+    }
```

Comment:
as we discussed, this may be the job of the parent. 

then again, it is nice to have a definition, for a listing of ipfs apps one can use. 

(i can see benefits of both sides, i guess. @diasdavid ?)